model-optimizer/extensions/front/mxnet/activation.py

```diff
@@ -44,3 +44,33 @@ def extract(cls, node):
                 act_type)
         act_class.update_node_stat(node)
         return cls.enabled
+
+
+class AsinhFrontExtractor(FrontExtractorOp):
```

Comment:
I don't see in the MXNet section of the  `docs/MO_DG/prepare_model/Supported_Frameworks_Layers.md` any of the added operations extractors. Also, it looks like this part of the document is significantly out of date. Please, update.